lib/logstash/inputs/imap.rb

```diff
@@ -111,11 +112,15 @@ def check_mail(queue)
       items = imap.uid_fetch(id_set, ["BODY.PEEK[]", "UID"])
       items.each do |item|
         next unless item.attr.has_key?("BODY[]")
-        mail = Mail.read_from_string(item.attr["BODY[]"])
-        if @strip_attachments
-          queue << parse_mail(mail.without_attachments!)
+        if @rfc822_mail
```

Comment:
Does it make sense to have the option to honour the `strip_attachments` config option here? If not, it should probably be documented that `strip_attachments` will be ignored if `rfc822_mail` is enabled.

Reply:
I think not. The purpose of `rfc822_mail` is to leave mail processing on filters.